site/src/routes/docs/_sections.js

```diff
@@ -93,7 +98,8 @@ export default function() {
 					slug = make_slug(rawtext);
 				}
 
-				if (level === 3 || level === 4) {
+				const tocIgnore = 'data-toc-ignore' in extracted.attrs;
+				if ((level === 3 || level === 4) && !tocIgnore) {
```

Comment:
i wonder what is this for?

Reply:
This is what determines what becomes a subsection in the table of contents. Or are you referring to something else?

Reply:
got it!